src/lib/cli/apiConfig.js

```diff
@@ -1,7 +1,5 @@
-/**
- * @flow
- * @format
- */
+// @flow
+// @format
```

Comment:
Ideally, this is what we have to do to all files abusing `@flow` and `@format` as JSDoc tags.